proto/proto/ntx_builder.proto

```diff
@@ -0,0 +1,17 @@
+// Specification of the ntx-builder RPC.
+//
+// This provided access to the rollup data to the other nodes.
+syntax = "proto3";
+package ntx_builder;
+
+import "requests.proto";
+import "responses.proto";
+import "google/protobuf/empty.proto";
+
+service Api {
+    // Submit a list of network notes to create network transactions.
+    rpc SubmitNetworkNotes(requests.SubmitNetworkNotesRequest) returns (google.protobuf.Empty) {}
```

Comment:
I would write the comment as "Submit a list of network notes to the network transaction builder."